shapes/profile-shape.ttl

```diff
@@ -0,0 +1,107 @@
+PREFIX sh: <http://www.w3.org/ns/shacl#>
+PREFIX dct: <http://purl.org/dc/terms/>
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX solid: <http://www.w3.org/ns/solid/terms#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+PREFIX org: <http://www.w3.org/ns/org#>
+PREFIX vcard: <http://www.w3.org/2006/vcard/ns#>
+PREFIX schema: <http://schema.org/>
+PREFIX foaf: <http://xmlns.com/foaf/0.1/>
+PREFIX : <#>
+
+:ProfileShape     a sh:NodeShape;
+    dct:title "Profile shape";
+    :property 
+            [
+        :datatype xsd:string;
+        :name "his/hers/theirs...";
```

Comment:
I think these are more examples, for in a description, rather than a name.
```suggestion
        sh:name "Preferred Relative Pronoun";
        sh:description "E.g. his/hers/theirs...";
```
(Same goes for the other pronoun properties.)